docs/v2.4/reference/config.md

```diff
@@ -92,19 +92,26 @@ These fields control how the client interacts with the Cloud service.
 
 | Field             | Description                                                                                                                                                                                                                                | Type                                       | Default |
 |-------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------|---------|
-| `skipLogin`       | Whether the cli should skip automatic login to Ambassador Cloud. If set to true, you must have a [license](../cluster-config/#air-gapped-cluster) installed in the cluster in order to be able to perform selective intercepts             | [bool][yaml-bool]                          | false   |
+| `skipLogin`       | Whether the CLI should skip automatic login to Ambassador Cloud.  If set to true, in order to perform personal intercepts you must have a [license key](../cluster-config/#air-gapped-cluster) installed in the cluster.                   | [bool][yaml-bool]                          | false   |
 | `refreshMessages` | How frequently the CLI should communicate with Ambassador Cloud to get new command messages, which also resets whether the message has been raised or not. You will see each message at most once within the duration given by this config | [duration][go-duration] [string][yaml-str] | 168h    |
 
-Telepresence attempts to auto-detect if the cluster is air-gapped,
-be sure to set the `skipLogin` value to `true`
+Telepresence attempts to auto-detect if the cluster is capable of
+communication with Ambassador Cloud, but may still prompt you to log
```

Comment:
Super nit so can ignore: Perhaps change `log in in` to `login in` so we don't have the `in` repetition.

Reply:
"login" is the noun form, "log in" is the verb form.